### PR TITLE
Use PrimaryToolbar for PoliciesTable

### PR DESCRIPTION
**Pull request by bastilian**
This changes the custom toolbar for the `PoliciesTable` to use the `PrimaryToolbar`.

Screenshot 2020-03-28 at 23 24 05

Screenshot 2020-03-28 at 23 23 52

It also removes the filter utilities to use the modules exported from `frontend-components-inventory-compliance` in https://github.com/RedHatInsights/frontend-components/pull/425

**Note:** Tests will fail until  https://github.com/RedHatInsights/frontend-components/pull/425 is merged and a new package version is released.

### CODE REVIEW

File: src/SmartComponents/PoliciesTable/PoliciesTable.js [hunk 1]
```diff
@@ -1,30 +1,19 @@
...
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components';
 import CreatePolicy from '../CreatePolicy/CreatePolicy';
 import DeletePolicy from '../DeletePolicy/DeletePolicy';
-import routerParams from '@redhat-cloud-services/frontend-components-utilities/files/RouterParams';
 import { paths } from '../../Routes';
-import debounce from 'lodash/debounce';
-import {
-    Link
-} from 'react-router-dom';
+import { stringToId } from 'Utilities/TextHelper';
```
Comment (akofink) [inline src/SmartComponents/PoliciesTable/PoliciesTable.js:16]:
```suggestion
```

File: src/SmartComponents/CompliancePolicies/CompliancePolicies.js [hunk 2]
```diff
@@ -32,26 +35,39 @@ const QUERY = gql`
 `;
 
 export const CompliancePolicies = () => {
-    const { data, error, loading, refetch } = useQuery(QUERY, { fetchPolicy: 'cache-and-network' });
+    const { data, error, loading, refetch } = useQuery(QUERY, { fetchPolicy: 'network-only' });
```
Comment (dLobatog) [inline src/SmartComponents/CompliancePolicies/CompliancePolicies.js:38]: Why not rely on the cache for the initial render and now it's network only?

File: src/SmartComponents/CompliancePolicies/CompliancePolicies.js [hunk 3]
```diff
@@ -32,26 +35,39 @@ const QUERY = gql`
...
             <PageHeader style={{ paddingBottom: '22px' }} className={ beta ? 'beta-page-header' : 'stable-page-header' } >
                 <PageHeaderTitle title="Compliance policies" />
-                { !loading && !beta && <ComplianceTabs/> }
+                { !beta && <ComplianceTabs/> }
             </PageHeader>
             <Main>
-                { loading ?
-                    <LoadingPoliciesTable /> :
-                    <PoliciesTable onWizardFinish={() => refetch()} policies={data.profiles.edges.map(profile => profile.node)} />
+                { policies && policies.length === 0 ?
```
Comment (dLobatog) [inline src/SmartComponents/CompliancePolicies/CompliancePolicies.js:64]: Note: this fixes 555 - it's an alternative to https://github.com/RedHatInsights/compliance-frontend/pull/487

File: src/SmartComponents/CompliancePolicies/CompliancePolicies.test.js [hunk 4]
```diff
@@ -29,7 +29,7 @@ describe('CompliancePolicies', () => {
                         }
                     }]
                 }
-            }, error: false, loading: false }));
+            }, error: undefined, loading: undefined }));
```
Comment (dLobatog) [inline src/SmartComponents/CompliancePolicies/CompliancePolicies.test.js:32]: Good point :sweat_smile:

File: src/SmartComponents/CompliancePolicies/CompliancePolicies.js (hunk 2 above)
Comment (bastilian) [reply src/SmartComponents/CompliancePolicies/CompliancePolicies.js:38]: Oh good catch, this is must have been introduced with a rebase-conflict resolving. Will change it back to cache-and-network.

**Comment by bastilian** [comment]
@dLobatog Thank you for reviewing! The fetchPolicy is now back to cache-and-network. 

As mentioned in the PR description the tests will fail until https://github.com/RedHatInsights/frontend-components/pull/425 is merged and released. Locally the test are already passing. 
It should just be a matter of bumping the version either manually or rebasing ontop of master with the new version. 🤞

**Comment by bastilian** [comment]
@akofink @dLobatog @redhat-cloud-services/frontend-components-inventory-compliance is now bumped and tests pass now.

**Pull request merged**